### PR TITLE
EVG-2636 Add Extra logging to Github webhooks/patch cancellation.

### PR DESCRIPTION
**Pull request by richardsamuels**

### CODE REVIEW

File: rest/route/github_routes.go [hunk 1]
```diff
@@ -118,6 +135,17 @@ func (gh *githubHookApi) Execute(ctx context.Context, sc data.Connector) (Respon
 					Message:    err.Error(),
 				}
 			}
+			grip.Info(message.Fields{
+				"source":    "github hook",
+				"msg_id":    gh.msgID,
+				"event":     "pull_request",
+				"action":    *event.Action,
```
Comment (bsamek) [inline rest/route/github_routes.go]: Could any of these be nil?

Comment (richardsamuels) [reply rest/route/github_routes.go]: Nope, NewGithubIntent verifies that.

Comment (bsamek) [reply rest/route/github_routes.go]: sgtm

File: rest/data/patch.go [hunk 2]
```diff
@@ -100,10 +98,6 @@ func (p *DBPatchConnector) AbortPatchesFromPullRequest(event *github.PullRequest
 	err = model.AbortPatchesWithGithubPatchData(*event.PullRequest.ClosedAt,
 		owner, repo, *event.Number)
 	if err != nil {
-		grip.Error(message.Fields{
```
Comment (bsamek) [inline rest/data/patch.go]: why remove this?

Comment (richardsamuels) [reply rest/data/patch.go]: The route that calls this logs this failure in far greater detail.

**Pull request merged**